pom.xml

```diff
@@ -85,13 +85,10 @@
        [${javac.version})
     for an open-ended enforcement
     -->
-    <enforced.java.version>[${javac.version},)</enforced.java.version>
```

Comment:
The comment about `java version enforced by the maven enforcer` should be removed, too.

Reply:
The property is removed, but the comment is still there:

https://github.com/apache/ozone/blob/3af5b7275d1c4dc16bfb46807766c2cbcfcd6173/pom.xml#L83-L87